source/python/idetect/model.py

```diff
@@ -279,6 +279,22 @@ class Location(Base):
         'Report', secondary=report_location, back_populates='locations')
 
 
+class TermType:
+    PERSON_TERM = 'person_term'
+    PERSON_UNIT = 'person_unit'
+    STRUCTURE_TERM = 'structure_term'
+    STRUCTURE_UNIT = 'structure_unit'
+    ARTICLE_KEYWORD = 'article_keyword'
+
+
+class ReportKeyword(Base):
+    __tablename__ = 'report_keyword'
+
+    id = Column(Integer, primary_key=True)
+    description = Column(String)
+    term_type = Column(String)
```

Comment:
I had originally called it ReportTerm before realizing that I already have a ReportTerm class.

However I am going to leave ReportTerm to refer to the official terms from IDMC, and keep these as keywords (with a keyword_type) as effectively that is how I am using them